### DISCUSSION TITLE
My BlueRetro doesnt work on my ps2

### DISCUSSION THREAD

**Discussion by Pessoinha22** [darthcloud/BlueRetro#1045] [Q&A]
I made a blueretro adapter using an esp32 myself, watched a bunch of videos on how to do it and it works fine if its unplugged from my ps2 (the controllers get recognized and i can configure it through the configuration page) I tried a bunch of firmware versions and still couldnt get it to work. Sometimes when I plug it into my ps2 the controllers dont even pair.

**Comment by ca7bc0c5f2** [comment]
Triple check your wiring and post some photos if you can. Did you use the PCB in the project to make the adapter?

I experienced similar issues with a tg16 dongle and discovered I had it miswired.

**Reply by Pessoinha22** [reply]
I already done that, looked the wiring scheme in 3 different youtube videos they were all the same. And I dindnt use the adapter for the project, I am just using some wires and a normal esp32, I will be installing the mod internally in my ps2, thats why I didnt use the pcb. (There are no wires in the esp32 because I already dessolderead them)
![20240725_180554](https://github.com/user-attachments/assets/70aaeca8-a22e-42ea-b425-ccc157a8200a)

**Reply by ca7bc0c5f2** [reply]
I may not have much more to add. If you’re trying to build hw2 I have not done that as it seems beyond my capabilities.

https://github.com/darthcloud/BlueRetro/wiki/BlueRetro-HW2-Internal-Install-Specification

if you’re trying to build hw1 the schematic below might help. In my case, I solely wired 8 volts (no 3.3). Given the lack of physical controller detection on this build, make sure nothing else is connected to the port. I had no issues with my memcard pro, but for testing you may want to disconnect all memory cards as well. I believe they share the controller bus.

https://github.com/darthcloud/BlueRetro/wiki/BlueRetro-Cables-Build-Instructions

Could you share which esp32 pins you’re connecting to which pins in the PS2 controller plug? Your esp32 may have them labeled on the back side.

**Reply by ca7bc0c5f2** [reply]
Also, which model PS2 is this? Thinking certain slims could be an issue. I know I needed to do a mod to get the first gen memcard pro to work in mine. I think it was a voltage issue there. Maybe a side effect of that fix cause my blueretro adapter to work without a hitch.

**Reply by Pessoinha22** [reply]
It is a 9000 series motherboard, I was using this schematic. I will take a closer look at the schematics in the link, thank you for the help so far.
![Uploading 20240726_160021.jpg…]()

**Comment by NosAlu** [comment]
Ese ESP32 que enviaste en la foto no te va a funcionar para la PS2
Comprate el DEVKIT V1 en todos los videos usan ese (tiene la numeracio de pines y esa es la diferencia mas notable) 
![20240723112742](https://github.com/user-attachments/assets/c9899a5e-42ba-476a-97d5-ab7b250cfdff)

**Comment by Pessoinha22** [comment]
Guys. I got it working, but there is a problem, none of the analog sticks work, I tried with a ps4 and a ps5 controller, any sugestions what it could be?

**Reply by FallenArzz** [reply]
How did you got it working? I'm having same issue with the same esp32-wroom-32D USB C Version.
